db/external_sst_file_ingestion_job.cc

```diff
@@ -793,7 +793,20 @@ Status ExternalSstFileIngestionJob::GetIngestedFileInfo(
     file_to_ingest->smallest_internal_key.SetFrom(key);
 
     iter->SeekToLast();
```

Comment:
Maybe move this to the else clause when it's not PlainTable?

Reply:
This is a good idea. Thanks! Already fixed it.